lib/src/file_picker_bottom_sheet.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+
+import 'package:misc_utilities/src/bottom_sheet_builder.dart';
+import 'package:misc_utilities/src/string_constant.dart';
+
+/// [FilePickerBottomSheet] is used to show the bottomsheet for selecting the
+/// [ImageSource] to use when selecting a file to upload.
+///
+/// [pickCamera] function is triggered when the camera [ListTile] is tapped,
+/// and it opens the camera to take a picture of the image
+///
+/// [pickGallery] function is triggered to allow the user select an image from
+/// their gallery.
+///
+/// `Navigator.of(context).pop()` is callded after [pickCamera] and [pickGallery]
+/// to close the bottomsheet automatically
+class FilePickerBottomSheet extends BottomSheetBuilder {
```

Comment:
I would propose renaming this widget to `ImagePickerBottomSheet` to make it less ambiguous

Reply:
Well noted. Renaming..